src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -643,6 +643,8 @@ slots:
       To make processing this field as unambiguous as possible, we recommend using 
       wikidata CURIEs, but the type of this field is deliberately unspecified.
     range: string
+    annotations:
+      propagated: true
```

Comment:
We dont need to add 

```
instantiates: sssom:Propagatable
```
 
to the slot definition?

Reply:
Weirdly enough, it seems to work (at least from the point of view of the code generator) even without it, but yes, I think that for correctness we need that.